src/test/java/com/wikia/webdriver/elements/mercury/components/Navigation.java

```diff
@@ -6,6 +6,7 @@
 import com.wikia.webdriver.common.skin.Skin;
 import com.wikia.webdriver.common.skin.SkinHelper;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.JoinTodayPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.notifications.Notifications;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.notifications.Notifications'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)